ts/react-responsive-components.tsx

```diff
@@ -119,7 +119,11 @@ export class ResponsiveRoot extends React.Component<ResponsiveRootProps, void> {
 export interface ResponsiveProps extends MixedInHoCProps {
 	currentBreakpoint?: Breakpoint
 	showAtOrAbove?: string,
-	showAtOrBelow?: string
+	showAtOrBelow?: string,
```

Comment:
I am wondering if we can split the props into two mutually exclusive types, one with showAtOrAbove and showAtOrBelow and another type with showBetween